examples/duel/src/main/webapp/js/todos/controller.js

```diff
@@ -85,20 +57,28 @@ var todos = todos || {};
 		} else {
 			todos.model.remove(id);
 		}
-		refreshAll();
+		refreshView();
+	}
+
+	function reset(input, id) {
+		var task = todos.model.find(id);
+		if (task) {
+			input.value = task.title;
+		}
 	}
 
 	/*-- export public interface -------------------------------*/
 
 	// event handlers
 	todos.actions = {
-		addBlur: function () {
-			add(this);
-		},
-
 		add_keypress: function (e) {
-			if (e.keyCode === ENTER_KEY) {
-				add(this);
+			switch (e.keyCode) {
```

Comment:
I personally think that for `keyCode` blocks a switch makes more sense. By the time you get a few then it becomes easier to just read the case labels than walking the if-else chain.

Arguably only two can go either way. Easy to switch back.